### PR TITLE
add support for multi gopath

### PR DESCRIPTION
**Pull request by perfectnewer**
I have set multi path for my `GOPATH` , so i think maybe someone else need too.

### CODE REVIEW

File: cmd/localInstall.go [hunk 1]
```diff
@@ -48,12 +51,20 @@ After that a 'vg ensure' will install like normal again.
...
-				utils.PkgToDir(pkg),
-			)
+			for _, gopath := range strings.Split(utils.OriginalGopath(), ":") {
+				pkgPath := filepath.Join(gopath, "src", utils.PkgToDir(pkg))
+				_, err := os.Stat(pkgPath)
+				if err == os.ErrNotExist {
+					continue
+				}
+				if err == nil {
+					path = pkgPath
```
Comment (JelteF) [inline cmd/localInstall.go]: Should break on the first found path

File: cmd/localInstall.go [hunk 2]
```diff
@@ -48,12 +51,20 @@ After that a 'vg ensure' will install like normal again.
 		if len(args) == 2 {
 			path = args[1]
 		} else {
-
-			path = filepath.Join(
-				utils.OriginalGopath(),
-				"src",
-				utils.PkgToDir(pkg),
-			)
+			for _, gopath := range strings.Split(utils.OriginalGopath(), ":") {
```
Comment (JelteF) [inline cmd/localInstall.go]: Should use `filepath.SplitList` instead for windows support: https://golang.org/pkg/path/filepath/#SplitList

File: cmd/localInstall.go (hunk 1 above)
Comment (perfectnewer) [reply cmd/localInstall.go]: sorry for my crealess.

File: cmd/localInstall.go (hunk 2 above)
Comment (perfectnewer) [reply cmd/localInstall.go]: fixed

**Pull request merged**